prow/config/config.go

```diff
@@ -77,6 +77,9 @@ type JobConfig struct {
 
 	// Periodics are not associated with any repo.
 	Periodics []Periodic `json:"periodics,omitempty"`
+
+	// AllRepos contains all Repos that have one or more jobs configured
```

Comment:
nit: This is the union of repos that jobs are on for and those `tide` is watching

Reply:
Updated the godoc